File: Problem Solving/Compare the Triplets/compareTriplets.js
//url : https://www.hackerrank.com/challenges/compare-the-triplets/problem

function compareTriplets(a, b) {
    // Write your code here
    var alice = 0;
    var bob = 0;
    var result = Array();
    for(var i=0;i<a.length;i++){
        bob += a[i] > b[i] ? 1 : 0;
        alice += a[i] < b[i] ? 1 : 0;
    }
    result.push(bob, alice);
    return result;
}

File: Problem Solving/Compare the Triplets/compareTriplets.go
//url : https://www.hackerrank.com/challenges/solve-me-first/problem

func compareTriplets(a []int32, b []int32) []int32 {
    // Write your code here
    bob := 0
    alice :=0      
    for i := 0; i < len(a); i++ {
        if(a[i] > b[i]){ bob = bob + 1}
        if(a[i] < b[i]){ alice = alice + 1}        
    }
    result := []int32{int32(bob), int32(alice)}    
    return result
}